src/main/kotlin/sc/iview/ui/SwingNodePropertyEditor.kt

```diff
@@ -339,21 +341,39 @@ class SwingNodePropertyEditor(private val sciView: SciView) : UIComponent<JPanel
                 val uiDebug = sciView.hub.get<Settings>()?.get("sciview.DebugUI", false) ?: false
 
                 inputPanel = harvester.createInputPanel()
-                inputPanel.component.layout = MigLayout("fillx,wrap 1,${if(uiDebug) "debug," else { "" } }insets 0 0 0 0", "[right,fill,grow]")
+                inputPanel.component.layout = MigLayout("fillx,wrap 1,insets 0 0 0 0".maybeActivateDebug(uiDebug), "[right,fill,grow]")
+
+                fun changeFont(component: Array<Component>, scale: Float) {
+                    component.forEach {
+                        val newFont = it.font.deriveFont(it.font.size * scale)
+                        it.font = newFont
+
+                        if(it is Container) {
```

Comment:
Do Container types have visible fonts themselves or are they just meta objects for the hierarchy? If the second is the case, you could put lines 348-349 in an else clause?